src/app/query/ProductList.query.js

```diff
@@ -130,6 +131,12 @@ export class ProductListQuery {
             this._getProductThumbnailField(),
             this._getShortDescriptionField(),
             this._getAttributesField(isVariant),
+            ...(isVariant && getLinksFieldFromCart
+                ? [
+                    this._getProductLinksField()
```

Comment:
Looks like a mess. Please refactor. Why if `isVariant` and `getLinksFieldFromCart`? Why not just `getLinksFieldFromCart`?